src/commands/sites/create.js

```diff
@@ -19,7 +20,20 @@ class SitesCreateCommand extends Command {
 
     let name = flags.name
     if (!name) {
-      console.log('Choose a site name or leave blank for a random name. You can update later.')
+      const userName = await api.getCurrentUser()
+      const suggestions = [
+        `super-cool-site-by-${userName.slug}`,
+        `the-awesome-${userName.slug}-site`,
+        `${userName.slug}-makes-great-sites`,
+        `netlify-thinks-${userName.slug}-is-great`,
+        `the-great-${userName.slug}-site`,
+        `isnt-${userName.slug}-awesome`,
+        `${userName.slug}-blog`
+      ]
+      const siteSuggestion = sample(suggestions)
```

Comment:
Probably not a big thing, but not sure we need lodash.sample for this, it's just a one-liner. I guess I could check how big that is though.

Reply:
You read my mind. https://github.com/netlify/cli/pull/334#issuecomment-503194907

Reply:
So I had it as 
```javascript
const siteSuggestion = suggestions[Math.floor(Math.random()*suggestions.length)]
```
But then I thought someone would ask me to put it in a util. Then I looked and saw we didn't have any utils for the CLI like this so I thought rather than rewriting a random function myself  I could inlude lodash's method which has _almost_ the same exact implementation. https://github.com/lodash/lodash/blob/master/sample.js. It should only import this one file and it has a slightly better implementation.

Reply:
@sdras does that work for you? Or would you rather me put it into a util?